components/sections/Footer/index.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import Image from 'next/image';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import NavItem from '@/components/sections/NavItem';
+import { useSiteConfig } from '@/hooks/useSiteConfig';
+
+import styles from './index.module.css';
+
+const Footer: FC = () => {
+  const { footerLinks, socialLinks } = useSiteConfig();
+
+  return (
+    <div className={styles.footer}>
+      <div className={styles.left}>
+        {footerLinks.map(item => {
+          return (
+            <NavItem type="footer" href={item.link} key={item.link}>
+              <FormattedMessage id={item.text} />
+            </NavItem>
+          );
+        })}
+      </div>
+      <div className={styles.right}>
+        <NavItem type="footer" href="https://openjsf.org/">
+          &copy; OpenJs Foundation
+        </NavItem>
+        <div className={styles.social}>
+          {socialLinks.map(link => {
+            const navClass = classNames({
+              [styles.darkImage]: link.kind === 'dark',
+              [styles.lightImage]: link.kind === 'light',
+            });
+
+            return (
+              <NavItem
+                className={navClass}
+                key={link.icon}
+                href={link.link}
+                type="footer"
```

Comment:
IMO it might be better to set it as a target='_blank' for the links we provide externally. Also, I'm not sure if this component will be used here. Adds extra node;

- a
	- span
		- img

Reply:
We could debate that in the future. As currently the links on nodejs.org are not external.